x-pack/legacy/plugins/uptime/public/lib/helper/stringify_url_params.ts

```diff
@@ -5,6 +5,18 @@
  */
 
 import qs from 'querystring';
-import { UptimeUrlParams } from './url_params/get_supported_url_params';
+import { UptimeUrlParams } from './url_params';
 
-export const stringifyUrlParams = (params: Partial<UptimeUrlParams>) => `?${qs.stringify(params)}`;
+export const stringifyUrlParams = (params: Partial<UptimeUrlParams>, ignoreEmpty = false) => {
+  if (ignoreEmpty) {
+    Object.keys(params).forEach((key: string) => {
+      // @ts-ignore
```

Comment:
I'm not a huge fan of this but [it seems to be a problem](https://github.com/microsoft/TypeScript/issues/24243) a lot of people [have faced](https://github.com/microsoft/TypeScript/issues/15295), so I suppose we can just deal with it until someone finds a better solution or TS modifies the return behavior of `Object.keys`.